unit_tests/simple_imports.py

```diff
@@ -7,6 +7,6 @@ def import_parent():
 
 
 def import_distributor():
-    directory = parent_dir + r"\tools\data_distributor"
+    directory = os.path.join(parent_dir, 'tools', 'data_distributor')
```

Comment:
Make constants for the folder names here, as they may change in the next reorganization.

Reply:
I think I will leave this one up to Cole, because it looks like he is finding the parent directory using some os function. I wouldn't know what constant to make it.